test/e2e/config/docker-dev.yaml

```diff
@@ -1,123 +0,0 @@
----
-# E2E test scenario using local dev images and manifests built from the source tree for following providers:
```

Comment:
I'm dropping dev config, so we should not anymore put efforts in keeping it up-to-date